addons/binding/org.openhab.binding.mysensors/src/main/test/org/openhab/binding/mysensors/test/CacheTest.java

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
Hooray tests!

However they should be in separate project, see https://www.eclipse.org/smarthome/documentation/development/testing.html